andela/wg-tomatoes
# Reset Cache on muscle delete

## What does this PR do?
This PR fixes the bug where muscles appear in the muscle overview after delete. It as well fixes the bug where old muscle details appear in a muscle list after a muscle has been edited.

## Description of the task to be completed
After a muscle is deleted in the admin overview it appears in the muscle list. If a muscle is edited, its details in the muscle list are not updated. This was due to caching issues. Our task was to therefore reset the cache if a muscle is edited or deleted.

## How can this be manually tested?
   1. Launch the application
   2. Click Exercise drop down menu to reveal Muscles.
   3. Delete a muscle
   4. Click Exercise to revael Muscle overview
  5. The muscle will not appear in the list.

## Relevant pivotal tracker stories?
Bug #154240682
